open_rfps/uptest.md

```diff
@@ -0,0 +1,59 @@
+
+
+Total amount of hours(0+10+30+30+30): 100
+
+## Payment terms:
+USD price of each milestone to be converted into XOR at delivery. 
+Receivers XOR address: oFFniTXNGeA9b8W62e1gsvz6zHPAqH4MfdZ79JbbvWvNxsfNc
```

Comment:
Hi, this doesn't appear to be a SORA network address (addresses start with cn)

Reply:
I checked on  polkadot.js apps and it gave me this address, will recheck

Reply:
@WRRicht3r checked and updated the address